extra/checksums/process/process.factor

```diff
@@ -20,14 +20,15 @@ M: process-state dispose* ( process-state -- )
 M: process-state add-checksum-bytes ( process-state bytes -- process-state' )
     over proc>> stream-write ;
 
-: trim-hash ( str -- str' ) dup " *-" swap start head ;
+: trim-hash ( str -- str' )
+    dup empty? [ dup [ blank? ] find drop [ head ] when* ] unless ;
```

Comment:
@jonenst : I've reimplemented `trim-hash` using `blank?`. If there is a simpler way, please advise. I did not find a way to use `unless-empty` here.